app/domain/permissions.py

```diff
@@ -239,9 +239,10 @@ def check_actor_can_write_on_mission_over_period(
     ):
         _raise_authorization_error()
 
-    # 5. Check that the mission is not yet validated by an admin
-    if mission.validated_by_admin or (
-        for_user and mission.validated_by_admin_for(for_user)
+    # 5. Check that the mission is not yet manually validated by an admin
+    if mission.manually_validated_by_admin or (
+        for_user
+        and mission.validated_by_admin_for(user=for_user, only_manual=True)
```

Comment:
question 1: `mission.manually_validated_by_admin` et `for_user and mission.validated_by_admin_for(user=for_user, only_manual=True)` c'est pas la même chose ?

question 2: est-ce qu'il n'y a pas certains cas où la validation admin auto ne bloque pas ? Par exemple, si un salarié veut valider après une validation admin auto ? Ah non, ce ne sera plus possible.. Je laisse quand même la question au cas où

Reply:
1. si on était sûr que `for_user` existe on pourrait n'appeler que `validated_by_admin_for` mais c'est pas le cas donc j'ai préféré répliquer comme c'était avant et faire un équivalent de `validated_by_admin` en manual